packages/termux-tools/pkg

```diff
@@ -76,6 +76,7 @@ select_mirror() {
 	mirrors[34]="https://deb.kcubeterm.me/termux-main"
 	mirrors[17]="https://grimler.se/termux-packages-24"
 	mirrors[14]="https://termux.mentality.rip/termux-main"
+	mirrors[13]="https://termux.librehat.com/apt/termux-main"
```

Comment:
`35 + 34 + 17 + 14 + 13` != `100`

Value here is used for weight-based mirror selection. By having the max weight of `100`, values will show rough probability (in percents) of selection of specific mirror. So don't specify arbitrary values here, duplicate values are not acceptable either.

Reply:
Suggested to modify like `35`, `18`, `17`, `16`, `14`.